source/transfer_list.rst

```diff
@@ -657,5 +660,116 @@ such that the last ACPI table in this entry ends at offset
      - hdr_size
      - One or more ACPI tables.
 
+.. _optee_pagable_part_entry:
+
+OPTEE pagable part entry layout (XFERLIST_OPTEE_PAGABLE_PART)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This entry type holds the address of OPTEE pagable part.
```

Comment:
I am struggling to review this as it seems to be split over two commits? Is there a way to see what is going on?

What is a pagable part?

Reply:
I just did a sqush rebase. Now all changes are in one commit.
Definitions added for 'pagable part'.